lib/docs/filters/tailwindcss/clean_html.rb

```diff
@@ -0,0 +1,63 @@
+module Docs
+  class Tailwindcss
+    class CleanHtmlFilter < Filter
+      def call
+        css('div.sticky.top-0').remove
+        css('.sr-only').remove
+        css('#sidebar').remove
+
+        css('#nav ul li').each do |node|
+          link = node.css("a").attr('href').to_s
+          if link.include? "https://"
+            node.remove()
+          end
+        end
+
+        # Remove right sidebar
+        css('#content-wrapper > div > div.hidden.xl\:text-sm.xl\:block.flex-none.w-64.pl-8.mr-8 > div > div').each do |node|
+          node.remove
+        end
+
+        # Remove advert
+        css('#__next > div.py-2.bg-gradient-to-r.from-indigo-600.to-light-blue-500.overflow-hidden').each do |node|
+          node.remove
+        end
```

Comment:
Those CSS selectors seem very fragile and might break after the slightest style change. Can you find a more robust selector, probably making use of `:contains`?

Reply:
I am clearly not proficient in the HTML parser used (Nokogiri?). Kindly help provide the snippet needed to make the selector more robust, perhaps using the `:contains` as you suggested; based on the original verbose selector string I am using.

Reply:
With respect to 

> examples are not rendered in a useful way

This is also connected in a way to the first comment 

> useless CSS and other markup classes

I am aware that one of the main purpose of devdocs is to make all the documentations look and feel consistent. However, I really hope a kind of "balance" can be allowed for documentations regarding CSS frameworks/tools, such as TailwindCSS. These specific types of documentations depend on the tool's own css styles to make the examples meaningful.

In a similar example like Bootstrap, the examples are [easy to remove, thanks to the `.bd-example` wrapper class](https://github.com/freeCodeCamp/devdocs/blob/1c3ceca2d2d344edefe422f0b5789022bbe724ce/lib/docs/filters/bootstrap/clean_html_v4.rb#L12) for all examples. 

However, TailwindCSS does not have such examples containers wrappers similar to Bootstrap's described above. Therefore, [the workaround I did was to create some sort of "balance"](https://github.com/damms005/devdocs/commit/31233ce00576441985475bad7d0dc0cf3862c20f), such that the TailwindCSS examples will be rendered in a useful way (very important) without sacrificing the core principles of Devdocs simplicity and consistent look and feel. However, @MasterEnoc advised against this. Hence, the examples rendered in such a non-useful way. Given that this "balance" is disallowed (and I will strongly advise that this be re-considered), and there are no wrapper classes to strip off examples, kindly advise on how to proceed. (To review that "balance" I am talking about, you can simply checkout [this commit](https://github.com/damms005/devdocs/commit/31233ce00576441985475bad7d0dc0cf3862c20f) - sorry it is on another repo on my GitHub).

Regarding the "useless CSS and other markup classes", as I said, I really do not know much about Nokogiri, except I simply remove all css classes. However my Nokogiri skill is proficient enough to remove the screen-size specific utility classes (i.e. classes prefixed with `lg:`, `sm:`, etc.).

Kindly advise on how to proceed.

Reply:
You can still use the `_tailwindcss.scss` file to improve the style if needed. but remember to keep the documentation similar to the other ones. The previous `_tailwindcss.sccs` file had more than 100k lines of css.

Reply:
The previous `_tailwindcss.sccs` is largely an unminified version of the official tailwindcss style; and it is required to correctly display the examples. I have made necessary changes, removed some Tailwindcss official styles that conflicts with devdocs'. Kindly find the changes in the latest commit